app/konnect/org-management/teams-and-roles/manage.md

```diff
@@ -41,6 +41,8 @@ Edit team roles from the {% konnect_icon organizations %} [**Organization**](htt
 
 1. Open one of the entity tabs: **Services** or **Runtime Groups**.
 
+1. Select the region you want to add from the **View region** drop-down menu. Members assigned to this role can only access {{site.konnect_short_name}} objects in the selected region. 
```

Comment:
```suggestion
1. Select the region you want to add from the **View region** drop-down menu. Members assigned to this role can only access {{site.konnect_short_name}} objects in the selected region. 
```
Would it make sense to tag this optional? 

If I don't have an enterprise plan, do I still see the region selector but with only one region?